test/testruntime/test_runtime.go

```diff
@@ -69,7 +69,19 @@ func Initialize(workingDir, homeDir, binDir string) commands.TestCommands {
 	return runtime
 }
 
-// newRuntime provides a new test.Runner instance working in the given directory.
+// InitializeNoInitialCommit creates a fully functioning test.Runner in the given working directory,
+// including necessary Git configuration to make commits. Creates missing folders as needed.
+// Does not create an initial commit.
+// This is useful for scenarios that require testing the behavior of Git Town in a fresh repository.
+func InitializeNoInitialCommit(workingDir, homeDir, binDir string) commands.TestCommands {
```

Comment:
To avoid duplication of init logic, you could call `InitializeNoInitialCommit` in the `Initialize` function on line 63 above.